### PR TITLE
Bad cache filename should throw.

### PR DESCRIPTION
**Pull request by jzohrab**
Without the throw, a bad cache filename returns unexpected dates.

### CODE REVIEW

File: src/events/scraper/load-cache/_get-local-date-from-filename.js [hunk 1]
```diff
@@ -19,6 +19,9 @@ module.exports = function getLocalDateFromFilename (filename, tz='America/Los_An
    */
   file = file.substr(0, 24)
 
+  if (!/\d{4}-\d{2}-\d{2}t\d{2}_\d{2}_\d{2}\.\d{3}[Zz]/.test(file))
```
Comment (ryanblock) [inline src/events/scraper/load-cache/_get-local-date-from-filename.js:22]: Does not test the complete filename, only up through the key name, so we're still missing keyname, hash, ext + .gz; without the keyname and ext (and .gz) things will break.

Comment (jzohrab) [reply src/events/scraper/load-cache/_get-local-date-from-filename.js:22]: Right, what on earth was I thinking.  Thanks!

Comment (jzohrab) [reply src/events/scraper/load-cache/_get-local-date-from-filename.js:19]: @ryanblock I think we agreed that this was good to be merged as-is, with just the datetime parse check.

Conflicts resolved.

**Comment by ryanblock** [comment]
Superseded by: https://github.com/covidatlas/li/pull/39

**Pull request closed without merging**